apps/vscode/package.json

```diff
@@ -28,37 +28,19 @@
   ],
   "private": true,
   "engines": {
-    "vscode": "^1.66.0"
+    "vscode": "^1.75.0"
```

Comment:
The changes in this file are to resolve these problems:

![activation-events](https://github.com/quarto-dev/quarto/assets/12505835/860d39e5-49a0-4cc9-8855-a8e0406beabb)

VS Code now does [implicit activation](https://github.com/microsoft/vscode-docs/blob/vnext/release-notes/v1_74.md#implicit-activation-events-for-declared-extension-contributions) when your extension provides a language or command.